packages/tic-tac-toe-bot/.env-cmdrc

```diff
@@ -1,7 +1,5 @@
 {
   "development": {
-    "BASE_URL": "http://localhost:9000",
```

Comment:
Why take this out from development?

Reply:
It's being set manually in index.ts, but yeah, it actually makes more sense to just remove it from there. I'll push up a fix real quick.